test/spec/load-config.spec.js

```diff
@@ -0,0 +1,21 @@
+const { loadConfig } = require("../../lib");
```

Comment:
do `require("../..")` to it tests the same as when user does `require("electrode-confippet")`